api/household/householdRouter.js

```diff
@@ -42,7 +41,7 @@ router.post('/', (req, res, next) => {
 
 // PUT - Update household by ID
 router.put('/:id', (req, res, next) => {
-  DB.update('households', req.params.id, req.body)
+  Households.update()
     .then((editedHousehold) => {
       res.status(200).json({
         message: `Household ${req.params.id} updated`,
```

Comment:
Will `${req.params.id}` still work here after removing it from (the previous) line 45 above?

(Same question in `locationRouter.js` at (old) line 41 vs 44.)